server/grpc_framework_server.go

```diff
@@ -70,17 +70,19 @@ func NewGrpcFrameworkServer(config *ServerConfig) (*GrpcFrameworkServer, error)
 	})
 
 	// Setup authentication
-	for issuer := range config.Security.Authenticators {
-		log.Infof("Authentication enabled for issuer: %s", issuer)
-
-		// Check the necessary security config options are set. Authentication is enabled. Therefore,
-		// either RoleManager must be provided (this implies use of the default authZ)
-		// or explicit authZ interceptors must be provided
-		// or external authZ checker must be proviced (this implies use of external_authorizer.go)
-		if config.Security.Role == nil &&
-			(config.AuthZUnaryInterceptor == nil || config.AuthZStreamInterceptor == nil) &&
-			config.ExternalAuthZChecker == nil {
-			return nil, fmt.Errorf("must supply role manager when authentication is enabled and default authZ is used")
+	if config.Security.Authenticators != nil {
+		for _, issuerWithClientID := range config.Security.Authenticators.ListIssuersWithClientID() {
```

Comment:
I am not sure why we were iterating through the Authenticators since we are not using `issuer` in the check other than for logging. Just checking for non-nil `config.Security.Authenticators` should be enough?

Reply:
A nil check should have been enough, but I wanted to log the issuers and clientIDs registered on startup.